pkg/minikube/config/extra_options.go

```diff
@@ -65,6 +65,17 @@ func (es *ExtraOptionSlice) Exists(value string) bool {
 
 // Set parses the string value into a slice
 func (es *ExtraOptionSlice) Set(value string) error {
+
+	// Check doesn't start or end with an opening quotation.
+	if strings.HasPrefix(value, "“") || strings.HasSuffix(value, "“") {
+		return fmt.Errorf("invalid value: canot contain quotation: %q", value)
```

Comment:
Can you verify if this is actually true ? Could it be it hangs only if the there is One Qutation without closing ?


Do u mind posting the output of minikube fore and after this PR?

Reply:
My mistake: There is only hanging currently if at the end there is a quotation mark: 
minikube vishal$ ./out/minikube start --driver=docker --extra-config=etcd.client-cert-auth=false“
😄  minikube v1.18.1 on Darwin 10.14.6
✨  Using the docker driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🏃  Updating the running docker "minikube" container ...
🐳  Preparing Kubernetes v1.20.2 on Docker 20.10.3 ...
    ▪ etcd.client-cert-auth=false“

Reply:
There is already an error that shows up when there is a quote at the beginning too: 
./out/minikube start --driver=docker --extra-config=“etcd.client-cert-auth=false”
😄  minikube v1.18.1 on Darwin 10.14.6
✨  Using the docker driver based on existing profile
❗  These --extra-config parameters are invalid: [“etcd]

❌  Exiting due to MK_USAGE: Valid components are: [apiserver controller-manager scheduler etcd kubeadm kubelet kube-proxy]

So we need to just check the end of the string for a quote.

Reply:
./out/minikube start --driver=docker --extra-config=“etcd.client-cert-auth=false
😄  minikube v1.18.1 on Darwin 10.14.6
✨  Using the docker driver based on existing profile
❗  These --extra-config parameters are invalid: [“etcd]

❌  Exiting due to MK_USAGE: Valid components are: [apiserver controller-manager scheduler etcd kubeadm kubelet kube-proxy]

Reply:
I have preserved the initial errors and also added the error when there's a suffix quote at the end: 
 ./out/minikube start --driver=docker --extra-config=etcd.client-cert-auth=false”
Error: invalid argument "etcd.client-cert-auth=false”" for "--extra-config" flag: invalid value: cannot contain end quotation: "etcd.client-cert-auth=false”"